plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -341,7 +348,28 @@ public Instance startService(String namespace,
         }
     }
 
-    @Override
+    // sets the home directory for a user defined with CHE_USER
+    private void changeWorkingDirOwner(String containerId, LineConsumer machineLogger) throws MachineException {
+        String prepareCmd = "CHE_USER=`id -un`;"
+                + "sudo chown -R ${CHE_USER}:docker /home/user;"
```

Comment:
And user may not have home directory at all. And main process of container may have some  permissions problems because of some hardcoded permissions on FS in container but not in home folder.